server/pkg/user/members.go

```diff
@@ -13,50 +13,74 @@ var (
 )
 
 type Members struct {
-	members map[ID]Role
-	fixed   bool
+	users        map[ID]Role
+	integrations map[IntegrationID]Role
+	fixed        bool
 }
 
 func NewMembers() *Members {
-	m := &Members{members: map[ID]Role{}}
+	m := &Members{
+		users:        map[ID]Role{},
+		integrations: map[IntegrationID]Role{},
+	}
 	return m
 }
 
 func NewFixedMembers(u ID) *Members {
-	m := &Members{members: map[ID]Role{u: RoleOwner}, fixed: true}
+	m := &Members{
+		users:        map[ID]Role{u: RoleOwner},
+		integrations: map[IntegrationID]Role{},
+		fixed:        true,
+	}
 	return m
 }
 
-func NewMembersWith(members map[ID]Role) *Members {
-	m := &Members{members: map[ID]Role{}}
-	for k, v := range members {
-		m.members[k] = v
+func NewMembersWith(users map[ID]Role) *Members {
+	m := NewMembers()
+	for k, v := range users {
+		m.users[k] = v
 	}
 	return m
 }
 
-func NewFixedMembersWith(members map[ID]Role) *Members {
-	m := &Members{members: map[ID]Role{}, fixed: true}
-	for k, v := range members {
-		m.members[k] = v
+func NewFixedMembersWith(users map[ID]Role) *Members {
+	m := &Members{
+		users:        map[ID]Role{},
+		integrations: map[IntegrationID]Role{},
+		fixed:        true,
+	}
+	for k, v := range users {
+		m.users[k] = v
 	}
 	return m
 }
 
 func CopyMembers(members *Members) *Members {
-	return NewMembersWith(members.members)
+	m := NewMembersWith(members.users)
+	for k, v := range members.integrations {
+		m.integrations[k] = v
+	}
+	return m
+}
+
+func (m *Members) Users() map[ID]Role {
+	users := make(map[ID]Role)
+	for k, v := range m.users {
+		users[k] = v
+	}
+	return users
 }
 
-func (m *Members) Members() map[ID]Role {
-	members := make(map[ID]Role)
-	for k, v := range m.members {
-		members[k] = v
+func (m *Members) Integrations() map[IntegrationID]Role {
+	integrations := make(map[IntegrationID]Role)
+	for k, v := range m.integrations {
+		integrations[k] = v
 	}
-	return members
+	return integrations
```

Comment:
```suggestion
	return maps.Clone(m.Integrations)
```

maps is `golang.org/x/exp/maps`